modules/videoio/src/videoio_c.cpp

```diff
@@ -0,0 +1,144 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "precomp.hpp"
+
+#include "videoio_registry.hpp"
+
+using namespace cv;
+
+// Legacy C-like API
+
+CV_IMPL CvCapture* cvCreateCameraCapture(int index)
```

Comment:
Previous version of `cvCreateCameraCapture` allows to specify preferred backend as `backend*100+cameraID`. It looks like updated version ignore provided backend.